### PR TITLE
Add docs for new env vars in actions

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: docs/configuration/actions.md [hunk 1]
```diff
@@ -125,7 +125,14 @@ actions:
...
 
-Additionally, packit sets a few env vars for specific actions.
+Additionally, packit sets a few env vars for all the actions:
+
+`PACKIT_UPSTREAM_PROJECT_NAME` — set to the `upstream_package_name` value, if any, or empty string  
+`PACKIT_DOWNSTREAM_PROJECT_NAME` — set to the `downstream_package_name` value, if any, or empty string  
+`PACKIT_PROJECT_NAME` — the package name key for the `packages` dictionary in a monorepo project,
+otherwise the `downstream_package_name` value is used, if any, or the `upstream_package_name`  
+
+and other env vars for specific actions:
```
Comment (nforro) [inline docs/configuration/actions.md:135]:
```suggestion
Additionally, packit sets several environment variables for the actions:

`PACKIT_UPSTREAM_PACKAGE_NAME` — set to the `upstream_package_name` value, if any
`PACKIT_DOWNSTREAM_PACKAGE_NAME` — set to the `downstream_package_name` value, if any
`PACKIT_PACKAGE_NAME` — set to the package name key for the `packages` dictionary in a monorepo project, falls back to `downstream_package_name` or, if not set, to `upstream_package_name`

There are also action-specific environment variables:
```

File: docs/configuration/actions.md [hunk 2]
```diff
@@ -136,14 +143,23 @@ Additionally, packit sets a few env vars for specific actions.
 
 **create-archive**
 
-`PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)
+`PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)
```
Comment (nforro) [inline docs/configuration/actions.md:146]:
```suggestion
`PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)
```

File: docs/configuration/actions.md [hunk 3]
```diff
@@ -136,14 +143,23 @@ Additionally, packit sets a few env vars for specific actions.
...
 (e.g. when syncing upstream release downstream)
 
+**release synchronization actions** (`propose-downstream` and `pull-from-upstream`)  
+*post-upstream-clone*
+*pre-sync*
+*prepare-files*
+*create-patches*
+
+`PACKIT_DOWNSTREAM_REPO` — absolute path to the downstream repo working dir  
+`PACKIT_UPSTREAM_REPO` — absolute path to the upstream repo working dir
```
Comment (nforro) [inline docs/configuration/actions.md:161]:
```suggestion
`PACKIT_UPSTREAM_REPO` — absolute path to cloned upstream git repo
`PACKIT_DOWNSTREAM_REPO` — absolute path to cloned downstream git repo
```

**Review by lachmanfrantisek — APPROVED** [review]
Thanks a lot, Maja!

**Pull request merged**